src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AlleleFractionInitializer.java

```diff
@@ -130,34 +130,27 @@ public AlleleFractionInitializer(final AlleleFractionData data) {
     }
 
     private double estimateOutlierProbability(final AlleleFractionData data) {
-        final UnivariateObjectiveFunction objective = new UnivariateObjectiveFunction(outlierProbability ->
-                AlleleFractionLikelihoods.logLikelihood(state.shallowCopyWithProposedOutlierProbability(outlierProbability), data));
-        final SearchInterval searchInterval = new SearchInterval(0.0, MAX_REASONABLE_OUTLIER_PROBABILITY, state.outlierProbability());
-        return OPTIMIZER.optimize(objective, GoalType.MAXIMIZE, searchInterval, BRENT_MAX_EVAL).getPoint();
+        final Function<Double, Double> objective = outlierProbability ->
```

Comment:
It seems that the only advantage of breaking apart the `AlleleFractionState` is that we can avoid the issue of shallow copies referring to the same `AlleleFractionState.MinorFractions` list?